ux.symfony.com/assets/app-vue.js

```diff
@@ -0,0 +1,5 @@
+import {registerVueControllerComponents} from "@symfony/ux-vue";
```

Comment:
`import {registerVueControllerComponents} from '@symfony/ux-vue';`

Single quotes to keep it in line with the rest :)

Reply:
Good catch, thank you